code/components/rage-formats-x/include/convert/phBound_five_rdr3.h

```diff
@@ -213,11 +213,225 @@ static inline void fillPolyhedronBound(rdr3::phBoundPolyhedron* out, five::phBou
 	}
 }
 
-static inline uint8_t ConvertMaterialIndexrdr3(uint8_t index)
+static inline uint16_t GenerateMaterialIndex(char* materialName)
```

Comment:
Pushed some changes. I don't know if that's what you meant. i'm really not a c++ programmer ):

Reply:
That's looking good. Well you are on your way and now you got some more powerful knowledge :)

You can remove the `inline` in front of it, nowadays compilers don't really adhere to it, becoming more of a hint to other programmers of your intent. If this wasn't evaluated at runtime and would actually be inlined then it would more likely introduce bloated code then any performance gain, I presume.

Reply:
inline Removed Thanks (: